Packages/manifest.json

```diff
@@ -3,10 +3,9 @@
     "com.unity.package-manager-ui": "2.0.8",
     "com.unity.textmeshpro": "1.4.1",
     "com.unity.xr.arcore": "2.1.2",
-    "com.unity.xr.arfoundation": "1.5.0-preview.6",
```

Comment:
I think we should keep this, since the Unity project checked in here should be set up for optimal MRTK testing. Removing it may also affect the NuGet build

Reply:
added back